Marlin/src/inc/SanityCheck.h

```diff
@@ -2123,12 +2123,12 @@ static_assert(hbm[Z_AXIS] >= 0, "HOMING_BUMP_MM.Z must be greater than or equal
   #if !(PIN_EXISTS(NEOPIXEL) && NEOPIXEL_PIXELS > 0)
     #error "NEOPIXEL_LED requires NEOPIXEL_PIN and NEOPIXEL_PIXELS."
   #endif
-  #elif ENABLED(NEOPIXEL2_SEPARATE)
```

Comment:
would never trigger no matter if NEOPIXEL2_SEPARATE was enabled or not with an #elif